packages/apollo-server/README.md

```diff
@@ -63,25 +69,21 @@ server.listen().then(({ url }) => {
 });
 ```
 
-> Due to its human-readability, we recommend using [schema-definition language (SDL)](https://www.apollographql.com/docs/apollo-server/essentials/schema.html#sdl) to define a GraphQL schema but [a `GraphQLSchema` object from `graphql-js`](https://github.com/graphql/graphql-js/#using-graphqljs) can also be specified in place of `typeDefs` and `resolvers` using the `schema` property:
+> Due to its human-readability, we recommend using [schema-definition language (SDL)](https://www.apollographql.com/docs/apollo-server/essentials/schema.html#sdl) to define a GraphQL schema--[a `GraphQLSchema` object from `graphql-js`](https://github.com/graphql/graphql-js/#using-graphqljs) can also be specified instead of `typeDefs` and `resolvers` using the `schema` property:
 >
 > ```js
 > const server = new ApolloServer({
 >   schema: ...
 > });
 > ```
 
-Finally, start the server using `node index.js` and open your web-browser to the URL which is output on the console.
+Finally, start the server using `node index.js` and go to the URL returned on the console.
 
-For more details, check out the Apollo Server [Getting Started guide](https://www.apollographql.com/docs/apollo-server/getting-started.html) of the documentation, or for a more comprehensive understanding, see the [fullstack tutorial](https://www.apollographql.com/docs/tutorial/introduction.html).
-
-For questions, the [Apollo commuinty on Spectrum.chat](https://spectrum.chat) is a great place to get assistance.
```

Comment:
I think we should leave the link to Spectrum here, even though you've added it below.